### ISSUE TITLE
Why the normalize scalr of input transform is 0.5 but not 255*0.5 ?

### ISSUE THREAD

**Issue by neverneverendup** [deepinsight/insightface#2354]
I wonder why the preprocess normalize scalar is different betweet train and inference.

- Train :in insightface/recognition/arcface_torch/dataset.py
```
class MXFaceDataset(Dataset):
    def __init__(self, root_dir, local_rank):
        super(MXFaceDataset, self).__init__()
        self.transform = transforms.Compose(
            [transforms.ToPILImage(),
             transforms.RandomHorizontalFlip(),
             transforms.ToTensor(),
             transforms.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]),
             ])
        self.root_dir = root_dir
        self.local_rank = local_rank
```
- Inference:  in insightface/recognition/arcface_torch/inference.py
```
def inference(weight, name, img):
    if img is None:
        img = np.random.randint(0, 255, size=(112, 112, 3), dtype=np.uint8)
    else:
        img = cv2.imread(img)
        img = cv2.resize(img, (112, 112))
    img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
    img = np.transpose(img, (2, 0, 1))
    img = torch.from_numpy(img).unsqueeze(0).float()
    img.div_(255).sub_(0.5).div_(0.5)
    net = get_model(name, fp16=False)
    net.load_state_dict(torch.load(weight))
    net.eval()
    feat = net(img).numpy()
    print(feat)
```

Why the normalize scalr of input transform for trainning is 0.5 but not 255*0.5 ? @nttstar

**Comment by DongHan9722** [comment]
@neverneverendup  
In my opinion:
for training data preparation:
```
        self.transform = transforms.Compose(
            [transforms.ToPILImage(),
             transforms.RandomHorizontalFlip(),
             transforms.ToTensor(),
             transforms.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]),
             ])
```
after transforms.ToTensor() : image value is in range [0,1]
after  transforms.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5])) : image value is in range [-1,1]

for inference:
after img.div_(255).sub_(0.5).div_(0.5): image value is also in range [-1,1]
